### PR TITLE
Use old must gather live image when installing pre 4.13 versions

### PR DESCRIPTION
**Pull request by clacroix12**
Fixes: #7809

### CODE REVIEW

**Comment by clacroix12** [comment]
Tested with job: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/8155/

```
2023-06-14 11:08:23  15:08:23 - MainThread - ocs_ci.utility.reporting - INFO  - Setting live must gather image to: registry.redhat.io/odf4/ocs-must-gather-rhel8:v4.12
```

Deployment failed but the image pull for must-gather was successful. 

@petr-balogh FYI

**Review by petr-balogh — DISMISSED** [review]

**Review by shylesh — DISMISSED** [review]
+1

**Comment by clacroix12** [comment]
@petr-balogh @shylesh I had to fix a couple fstring tox errors before I could merge so I'll need approvals once more please

**Review by petr-balogh — APPROVED** [review]

**Pull request merged**
